ui/src/features/Editor/hooks.ts

```diff
@@ -379,10 +382,31 @@ export default ({
     (event: MouseEvent) => {
       if (yAwareness) {
         throttledMouseMove(event, yAwareness, screenToFlowPosition);
+        yAwareness.setLocalStateField("zoomLevel", getZoom());
```

Comment:
Does throttling the zoom cause any issues? Best would be moving this into the throttledMouseMove, and also add a conditional check to see if the zoom level has changed (if not, don't update yAwareness). 
Because currently we update zoom level EVERY time, which means every pixel I think since its not throttled. Also, even if zoom doesn't change, we try to set it. Which I think yjs might be smart enough to not do anything in that case, but better to avoid if possible.